policy/release/tasks.rego

```diff
@@ -197,11 +197,40 @@ current_required_tasks := task_data if {
 
 ## get the required task data for a pipeline with a label
 required_pipeline_task_data := task_data if {
-	some att in lib.pipelinerun_attestations
+	some att in _pipelinerun
 	count(tkn.tasks(att)) > 0
 	task_data := tkn.required_task_list(att)
 }
 
-_status(task) := status if {
+_status(task) = status if {
+	not task.status.conditions
 	status := task.status
+} else = status if {
+	some condition in task.status.conditions
+	condition.type == "Succeeded"
+	status := _slsav1_status(condition)
 } else = "MISSING"
+
+_slsav1_status(condition) := status if {
+	condition.status == "True"
+	status := "Succeeded"
+}
+
+_slsav1_status(condition) := status if {
+	condition.status == "False"
+	status := "Failed"
+}
+
+_pipelinerun := att if {
+	att := [a |
+		some a in lib.pipelinerun_attestations
+	]
+	count(att) > 0
+}
+
+_pipelinerun := att if {
+	att := [a |
+		some a in lib.pipelinerun_slsa_provenance_v1
```

Comment:
What if an image has both? Maybe something like this is safer?

```rego
v1_0 := [a |
  some a in lib.pipelinerun_slsa_provenance_v1
]
v0_1 := [a |
  some a in lib.pipelinerun_attestations
]

attestations := array.concat(v1_0, v0_1)
```

Reply:
This is a good question! I guess I was thinking one attestation per build, but I don't think there's any downside to this and we're protected in case it does happen.